config/crd/bases/ramendr.openshift.io_volumereplicationgroups.yaml

```diff
@@ -98,9 +98,6 @@ spec:
                 description: Desired state of all volumes [primary or secondary] in
                   this replication group; this value is propagated to children VolumeReplication
                   CRs
-                enum:
-                - Primary
-                - Secondary
```

Comment:
This is automatic change happened after the build. I chose not to change the automatic modifications.

Reply:
This happened because the shim [enforced](https://github.com/ShyamsundarR/volrep-shim-operator/blob/2f9f9b83efb60c6e91e9a39a2a1bbf1e60e56289/api/v1alpha1/volumereplication_types.go#L29) a set of valid values, and possibly VolRep addon does not.

This change as such is fine.

Reply:
I assume the reason is this (though not exactly sure). @ShyamsundarR do you think below reasoning is correct?

In VolRep, the states for primary and secondary are strings assigned as "primary" and "secondary" (i.e. withou capital letters for p and s). So, in the yaml file for VRG when state is given as "Primary" or "Secondary" it failed and giving small letters succeeded. 

It might be better for VolRep (may be even for VRG) to make a case insensitive string comparison of the states. But I guess that can be done in a separate PR?